intellij.md

```diff
@@ -0,0 +1,137 @@
+
+1. Öffnen Sie [GitHub](https://github.com/) und melden Sie sich mit Ihrem Account an.
+2. Öffnen Sie Ihre Profileinstellungen und navigieren Sie zu `Emails`.
+3. Fügen Sie hier ihre Email-Adresse der Hochschule Flensburg hinzu (endet auf @stud.hs-flensburg.de).
+4. Anschließend bekommen Sie von GitHub eine Bestätigungs-Email zugeschickt. Öffnen Sie diese und klicken Sie auf `Confirm email address`.
+5. Öffnen Sie GitHubs [Studierendenportal](https://education.github.com/) und melden Sie sich mit Ihrem GitHub-Account an.
+6. Klicken Sie auf `Join Global Campus` und anschließend auf `Yes, I'm a student`. 
+7. Wählen Sie in dem untenstehenden Formular Ihre Hochschul-Email-Adresse aus, schreiben Sie einen kurzen Text in das folgende Textfeld und klicken Sie auf `Continue`.
+8. Fügen Sie ein Bild Ihrer Studierendenbescheinigung bei und klicken Sie auf `Process my application`.
+    - Die Verifizierung durch GitHub dauert in der Regel nur wenige Minuten.
+9. Öffnen Sie nun JetBrains [Studierendenportal](https://www.jetbrains.com/shop/eform/students).
+10. Klicken Sie auf `GitHub` und anschließend auf `Mit GitHub autorisieren`.
+11. Melden Sie sich mit Ihrem GitHub-Account an und klicken Sie auf `Authorize`.
+12. Folgen Sie den Anweisungen des Formulars und klicken Sie auf `Kostenlose Produkte beantragen`
+13. Über die [Lizenzverwaltung](https://account.jetbrains.com/licenses) können Sie nun Ihr `JetBrains Product Pack for Students` aktivieren und alle Produkte von JetBrains herunterladen.
+
+## Import eines Projekts
+Den Import eines Projekts von GitHub können Sie entweder manuell vornehmen oder über die GitHub-Integration in IntelliJ durchführen.
+
+### Manuell
+1. Clonen Sie das Projekt in einen beliebigen Ordner auf Ihrem Rechner.
+2. Öffnen Sie IntelliJ und wählen Sie `File > New > Project from Existing Sources...`.
+3. An dieser Stelle kann ein "Import Project"-Dialog auftreten. Wählen Sie `Create project from existing sources` und bestätigen Sie mit `Next`.
+4. Navigieren Sie zu dem Ordner, in dem Sie das Projekt abgelegt haben, und wählen Sie diesen aus.
+5. Klicken Sie auf `Ok`.
+
+### GitHub Integration
+Einmalig: Navigieren Sie in IntelliJ zu `File > Settings > Version Control > GitHub` und setzen Sie den Haken bei `Clone git repositories using ssh`.
+1. Öffnen Sie IntelliJ und wählen Sie `File > New > Project from Version Control...`.
+2. Wählen Sie `Repository URL` aus und fügen Sie unter `URL` den SSH-Clone-Link des Repositories ein.
+   - Über das Feld `Directory` können Sie den Zielordner angeben, in dem das Projekt abgelegt werden soll.
+3. Klicken Sie auf `Clone` und anschließend auf `Trust Project`.
+
+### Hinweis
+Wenn Sie Ihr Projekt zuvor in Eclipse gestartet haben, kann es beim Import zu einem Konflikt mit der Import-Konfiguration kommen. 
+In einem Dialog müssen Sie auswählen, wie sie das Projekt öffnen möchten.
+Klicken Sie hier auf `Gradle project` und bestätigen Sie den Dialog mit `OK`. 
```

Comment:
Ich vermute, diese Beschreibung bezieht sich auf den Screenshot, den ich angehängt hatte. Die Bezeichnungen im Screenshot sind ja leicht anders als hier im Text. Hießen die Punkte bei dir wirklich so?

Reply:
Dein Screenshot bezog sich auf den Fall, dass das Projekt nach dem Pullen direkt in IntelliJ geöffnet wird. 

Dieser Abschnitt bezieht sich hingegen auf den Fall, dass ein Projekt zunächst in Eclipse angelegt wird und später in IntelliJ geöffnet wird. 

Der Dialog sieht dann bei mir folgend aus:

![image](https://user-images.githubusercontent.com/75450043/219709392-d2aedd6c-6694-4bae-8ca3-4b548f89deae.png)